ludwig/models/hyperopt.py

```diff
@@ -0,0 +1,76 @@
+#! /usr/bin/env python
+# coding=utf-8
+# Copyright (c) 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import logging
+from ludwig.utils.misc import get_from_registry
+
+
+logger = logging.getLogger(__name__)
+
+
+class RandomStrategy:
```

Comment:
Some of the previous comments were about these objects, i did not see them before doing the comments, sorry. So, instead of "reating" thsoe objects, they should be modified according to the previous indications, so they should extend the base strategy and executor class.

Reply:
Makes sense. Noted! @w4nderlust